os/net/Makefile

```diff
@@ -116,6 +116,7 @@ include lwip/sys/arch/Make.defs
 endif
 
 endif
+include utils/Make.defs
```

Comment:
including utils only when UDS is enabled. because network doesn't need it

Reply:
Fixed it